src/components/ExistingProofDialog.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <v-dialog>
+    <v-card>
+      <v-card-title>
+        {{ $t('ExistingProofDialog.ChooseLatestProofs') }} <v-btn style="float:right;" variant="text" density="compact" icon="mdi-close" @click="close"></v-btn>
+      </v-card-title>
+      <v-divider></v-divider>
+      <v-card-text>
+        <v-row>
+          <v-col cols="6" sm="6" md="3" v-for="proof in userProofList" :key="proof">
```

Comment:
```suggestion
          <v-col cols="12" sm="6" md="3" v-for="proof in userProofList" :key="proof">
```
On small screens I think we should just show the list in 1 column. Also avoids having the zoomed image when selected, not really needed as the user can zoom already with 2 fingers

Reply:
Alright